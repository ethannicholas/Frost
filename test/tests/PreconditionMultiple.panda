class PreconditionMultiple1 {
    @pre(x > 0)
    @pre(x % 2 = 1)
    method test(x:Int) {
    }
}

class PreconditionMultiple2 : PreconditionMultiple1 {
    @override
    @preOr(x < 0)
    @preOr(x % 2 = 0)
    method test(x:Int) {
    }
}
