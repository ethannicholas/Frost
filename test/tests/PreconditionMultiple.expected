PreconditionMultiple.panda:3:16: error: found more than one precondition
PreconditionMultiple.panda:11:18: error: found more than one precondition
2 errors
